packages/css/src/card.css

```
@@ -5,6 +5,9 @@
   --dsc-card-border-width: 1px;
   --dsc-card-border-style: solid;
   --dsc-card-border-color: var(--ds-color-border-subtle);
+  --dsc-card-block-border-width: 1px;
+  --dsc-card-block-border-style: solid;
+  --dsc-card-block-border-color: var(--ds-color-border-subtle);
```

Kunne disse evt. i utgangspunktet være 
```suggestion
  --dsc-card-block-border-width: var(--dsc-card-border-width);
  --dsc-card-block-border-style: var(--dsc-card-border-style);
  --dsc-card-block-border-color: var(--dsc-card-border-color);
```
...så er "arv" default om man ikke gjør noe annet? ☺️

---

Great idea :)